### ISSUE TITLE
aws-msk-alpha: Add Support for MSK Serverless

### ISSUE THREAD

**Issue by flo-mair** [aws/aws-cdk#28709]
### Describe the feature

Add support for MSK Serverless in alpha module. I'm aware that the CfnServerlessCluster construct exists in aws-cdk-lib/aws-msk. But the alpha module allows easier access to connection strings via `new CfnOutput(this, 'BootstrapBrokerStringSaslIam', { value: cluster.bootstrapBrokersSaslIam });` which would be great to have for Serverless as well without building a workaround with CR.

### Use Case

Easy access to connection strings.

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### CDK version used

2.121.1

### Environment details (OS name and version, etc.)

macOS

**Comment by pahud** [comment]
Yeah it would be great to have ServerlessCluster L2 construct in the alpha package.
